src/plugins/visualizations/public/visualize_app/utils/utils.ts

```diff
@@ -25,7 +25,7 @@ export const addHelpMenuToAppChrome = (chrome: ChromeStart, docLinks: DocLinksSt
     links: [
       {
         linkType: 'documentation',
-        href: `${docLinks.links.visualize.guide}`,
+        href: `${docLinks.links.dashboard.guide}`,
```

Comment:
@VriaA, as discussed with Florent, let's make this link also point to the same place:
```suggestion
        href: `${docLinks.links.visualize.guide}`,
```